### PR TITLE
CI: RPM build and Prompt for approval for untrusted PRs

### PR DESCRIPTION
**Pull request by mellery451**
c89f3ee has already been reviewed in https://github.com/ripple/rippled/pull/2494, but I needed to build on some of the changes there to add untrusted prompts. I'll close the other PR once this passes.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -69,15 +69,27 @@ try {
...
+"""
+                            ])
+                        )
+                    }
+                    catch (e) {
+                        echo 'had a problem interacting with github...comments are probably not updated'
+                    }
+
+                    input (
+                        message: "User $CHANGE_AUTHOR has submitted a PR #$CHANGE_ID. Okay to proceed with building?")
```
Comment (wilsonianb) [inline Jenkinsfile]: Should this include a reminder to review the pr (and what to keep an eye out for) before clicking Proceed?

**Comment by mellery451** [comment]
I've updated the prompt message slightly @wilsonianb. Here are two sample PRs built using these changes, one that was cancelled and one that was approved:

https://github.com/mellery451/rippled/pull/9
https://github.com/mellery451/rippled/pull/11

**Comment by JoeLoser** [comment]
Thanks for adding the part about prompting for approval to help with non Ripple devs contributing. I'm sure @seelabs didn't want to keep opening new PRs for me :)

**Comment by seelabs** [comment]
In 1.1.0-b2

**Pull request closed without merging**